src/Makefile

```diff
@@ -77,11 +83,8 @@ install: metadatad
 
 export DISTRO
 PKG_NAME=stackdriver-metadata
-PKG_VERSION=0.0.17
-PKG_RELEASE=1
 PKG_MAINTAINER=Stackdriver Agents <stackdriver-agents@google.com>
```

Comment:
How about moving all variables starting with PKG_ and DOCKER_ to the top so that they are defined in the same place?

Reply:
I was debating whether to do this. At this point, only versions (that will be changed with each subsequent release) live at the top. Adding more stuff there may be confusing to readers.

Reply:
Okay, that makes sense.